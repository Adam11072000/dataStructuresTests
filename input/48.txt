init 64
quit