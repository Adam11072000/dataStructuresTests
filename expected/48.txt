init done.
addImage: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
deleteImage: FAILURE
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: FAILURE
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
setLabelScore: FAILURE
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
deleteImage: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
deleteImage: INVALID_INPUT
setLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: FAILURE
getHighestScoredLabel: INVALID_INPUT
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
getHighestScoredLabel: INVALID_INPUT
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
resetLabelScore: FAILURE
setLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
getHighestScoredLabel: FAILURE
resetLabelScore: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
setLabelScore: FAILURE
setLabelScore: SUCCESS
getHighestScoredLabel: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: FAILURE
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
deleteImage: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: FAILURE
resetLabelScore: FAILURE
addImage: SUCCESS
setLabelScore: SUCCESS
addImage: INVALID_INPUT
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
addImage: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
mergeSuperPixels: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
resetLabelScore: FAILURE
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: INVALID_INPUT
addImage: SUCCESS
addImage: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: FAILURE
addImage: FAILURE
mergeSuperPixels: SUCCESS
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
addImage: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: FAILURE
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
addImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
setLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
getHighestScoredLabel: FAILURE
setLabelScore: SUCCESS
resetLabelScore: FAILURE
getHighestScoredLabel: FAILURE
mergeSuperPixels: INVALID_INPUT
resetLabelScore: FAILURE
resetLabelScore: INVALID_INPUT
setLabelScore: SUCCESS
setLabelScore: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
resetLabelScore: FAILURE
setLabelScore: SUCCESS
setLabelScore: FAILURE
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
mergeSuperPixels: SUCCESS
deleteImage: SUCCESS
addImage: SUCCESS
setLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
getHighestScoredLabel: FAILURE
setLabelScore: FAILURE
addImage: SUCCESS
mergeSuperPixels: SUCCESS
setLabelScore: INVALID_INPUT
quit done.
